core/completion.py

```diff
@@ -548,14 +553,14 @@ def __init__(self, mem):
     # (dir, timestamp) -> list of entries perhaps?  And then every time you hit
     # tab, do you have to check the timestamp?  It should be cached by the
     # kernel, so yes.
-    self.ext = []
+    # XXX(unused?) self.ext = []
```

Comment:
Ah right, forgot about this. `git grep "\.ext\."` suggests there aren't any users of this member. Should we drop it? If not, I'm not sure what the type should be... The comment above suggests it might be a list of some sort of dirent-like thing

Reply:
I don't remember what this is now, but yes let's just leave it out of this PR